tests/test_api.py

```diff
@@ -24,8 +29,8 @@ class TestApi:
     def test_pytket_to_phir_no_machine(self) -> None:
         """Test case when no machine is present."""
         circuit = get_qasm_as_circuit(QasmFile.baby)
-
-        assert pytket_to_phir(circuit)
+        phir = pytket_to_phir(circuit)
+        PHIRModel.model_validate_json(phir)
```

Comment:
I believe `phirgen` is already doing model validation, so to improve these tests the most we can do is assert for specific JSON entries in them. But they don't need to be done in this PR.

Reply:
Ah, thought it was only getting used sometimes, I see that it always does it.